src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.ServiceControlTransport.approved.txt

```diff
@@ -27,22 +19,15 @@ namespace ServiceControl.Transports
     public interface IProvideQueueLength
     {
         void Initialize(string connectionString, ServiceControl.Transports.QueueLengthStoreDto storeDto);
-        void Process(ServiceControl.Transports.EndpointInstanceIdDto endpointInstanceIdDto, string queueAddress);
-        void Process(ServiceControl.Transports.EndpointInstanceIdDto endpointInstanceIdDto, ServiceControl.Transports.TaggedLongValueOccurrenceDto metricsReport);
         System.Threading.Tasks.Task Start();
         System.Threading.Tasks.Task Stop();
+        void TrackEndpointInputQueue(string endpointName, string queueAddress);
     }
     public class QueueLengthStoreDto
     {
```

Comment:
Is there any value in `QueueLengthStoreDto` instead of `Action<EntryDto[], EndpointInputQueueDto>`?

Reply:
Removed even more, take a look now :)